Gemfile

```diff
@@ -20,7 +20,7 @@ gem 'mongoid', '6.1.0'
 gem 'mongoid-history', '0.6.1'
 gem 'mongoid_userstamp', git: 'https://github.com/DCAFEngineering/mongoid_userstamp.git', branch: 'master'
 gem 'mongo_session_store', '>= 3.1.0'
-gem 'mongoid-enum', git: 'https://github.com/DCAFEngineering/mongoid-enum', branch: 'master'
+gem 'enumerize'
```

Comment:
I believe it to be, they are quite active with >1000 stars (not sure if that's an accurate gauge).